wishlist.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Vacay Store / Wishlist</title>
+    <link href="https://vacay-ui.netlify.app/components/vacay.css" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="grid-container-wishlist">
+    <header class="header box-shadow">
+        <h1 class="logo">vacay</h1>
+        <div class="search-box">
+            <i class="fas fa-search"></i>
+            <input type="text" placeholder="Search">
+        </div>
+        <nav class="navigation">
+            <ul>
+                <li><a href="#" class="btn btn-primary">Login</a></li>
+                <li>
+                    <div class="badge-wrapper">
+                        <a href="#"><i class="btn-icon far fa-heart"></i></a>
```

Comment:
extra div can be removed. you have a wrapper of li can give the class="badge-wrapper" to li directly. Check if it works.

Reply:
Yes it works @vrx29 . I have removed the extra div. Thanks